src/ts/components/dashboard/Dashboard.tsx

```diff
@@ -1,19 +1,21 @@
 import * as React from 'react'
 import { getAccounts, lockWallet } from '../../services/keyring-vault-proxy'
-import { LOGIN_ROUTE } from '../../constants/routes'
+import { GENERATE_PHRASE_ROUTE, LOGIN_ROUTE } from '../../constants/routes'
 import { RouteComponentProps, withRouter } from 'react-router'
 import { Button as StyledButton, ContentContainer, Section } from '../basic-components'
 import { IAppState } from '../../background/store/all'
 import { connect } from 'react-redux'
-import { setCurrentAddressAndName } from '../../background/store/account'
```

Comment:
`setCurrentAccount` may be easier to read

Reply:
? it is now `setCurrentAccount `

Reply:
sorry, page not refreshed. All good